.github/workflows/indent.yml

```
@@ -2,23 +2,23 @@ name: Indent
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [master, develop]
   pull_request:
-    branches: [ master, develop ]
+    branches: [master, develop]
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: DoozyX/clang-format-lint-action@v0.11
-      with:
-        source: '.'
-        exclude: ''
-        extensions: 'h,cc'
-        clangFormatVersion: 8
-        inplace: true
-    - run: |
-        git diff
-        git diff-files --quiet || exit $?
+      - uses: actions/checkout@v2
+      - uses: DoozyX/clang-format-lint-action@v0.11
+        with:
+          source: "."
+          exclude: "./src/json.hpp"
+          extensions: "hpp,cpp,c,h"
+          clangFormatVersion: 8
```

```suggestion
          clangFormatVersion: 12
```
We will upgrade the clang format version in preCICE soon to version 12. I would also do this here.

---

Let's early adapt